src/utils.js

```diff
@@ -496,7 +496,7 @@ function isProductionMode(loaderContext) {
   return loaderContext.mode === "production" || !loaderContext.mode;
 }
 
-const defaultMinimizerOptions = {
+export const defaultMinimizerOptions = {
```

Comment:
Let's reexport it in `index.js`, we should avoid using `html-loader/dist/utils`, because structure can be changed, better to use `require('html-loader').defaultMinimizerOptions`

Reply:
reexported + fixed docs